docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.                #NOTE: Also, no need to add the T_i itself to the candidates. Because, ... 
</pre><p>I compared this to the original MERLIN paper and I noticed that they did not do this (Table 1?). So, their implementation contains these additional candidates. Is that correct?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I think they actually considered it (see line 9 of table 1). The `iscandidate` variable becomes `false` when its distance to (at least) one of the so-far-selected candidates become less than `min_dist` (see line 5-8). In that case, the current subsequence is not going to be added to the candidate set.

Reply:
<p>Got it. Thanks</p>